pints/toy/_toy_classes.py

```diff
@@ -61,3 +79,71 @@ def suggested_times(self):
         """
         raise NotImplementedError
 
+    def _rhs(self, y, t, p):
+        """
+        Returns RHS of ODE for numerical integration to obtain outputs and
+        sensitivities. This should be a vector of length ``n_outputs``.
+        """
+        raise NotImplementedError
+
+    def _rhs_S1(self, y_and_dydp, t, p):
+        """
+        Forms the RHS of ODE for numerical integration to obtain both
+        outputs and sensitivities. This should be a vector of length
+        ``n_outputs`` + ``n_parameters``.
+        """
+        y = y_and_dydp[0:self.n_outputs()]
```

Comment:
maybe can make this a bit more readable by using temp variables for n_outputs and n_parameters?
```python
n = self.n_outputs()
m = self.n_parameters()
```

Reply:
Done